### PR TITLE
RMET-2564/5 ::: Bridge ::: Add EditURIPicture.

### PR DESCRIPTION
**Pull request by OS-ricardomoreirasilva**
## Description
Include the new `EditURIPicture` to the `Camera.js` file. This will bridge the Outsystems implementation to the Android/iOS native libraries.

## Context
https://outsystemsrd.atlassian.net/browse/RMET-2564
https://outsystemsrd.atlassian.net/browse/RMET-2565

## Type of changes
- [ ] Fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Refactor (cosmetic changes)
- [ ] Breaking change (change that would cause existing functionality to not work as expected)

## Platforms affected
- [ ] Android
- [ ] iOS
- [x] JavaScript

## Checklist
- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [x] Code follows code style of this project
- [x] CHANGELOG.md file is correctly updated
- [x] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

**Review by alexgerardojacinto — APPROVED** [review]

**Review by nflsilva — APPROVED** [review]

File: www/Camera.js [hunk 1]
```diff
@@ -200,6 +196,19 @@ cameraExport.cleanup = function (successCallback, errorCallback) {
     exec(successCallback, errorCallback, 'Camera', 'editPicture', args);
 };
 
+cameraExport.editURIPicture = function (successCallback, errorCallback, options) {
+    argscheck.checkArgs('fFO', 'Camera.editURIPicture', arguments);
+    options = options || {};
+    
+    let uri = options.uri;
+    let saveToGallery = !!options.saveToGallery;
+    let includeMetadata = !!options.includeMetadata;
```
Comment (hectorjdgdev) [inline www/Camera.js]: would make sense to use destructure also here something like 
let {uri, saveToGallery, includeMetadata} = options

Comment (OS-ricardomoreirasilva) [reply www/Camera.js]: The idea was to follow a similar approach to other methods, but if it works, I don't see any reason not to use it

Comment (OS-ricardomoreirasilva) [reply www/Camera.js]: Fixed! Can you check again?

**Review by hectorjdgdev — APPROVED** [review]

File: www/Camera.js (hunk 1 above)
Comment (Chuckytuh) [reply www/Camera.js]: is this new version of camera plugin supposed to work for MABS 5 targets? If so, are you sure those Android/iOS OS versions support both let and destructuring?

Comment (OS-ricardomoreirasilva) [reply www/Camera.js]: We're not targeting MABS 5 on these developments.

**Pull request merged**